packages/mdc-select/_mixins.scss

```diff
@@ -101,8 +101,7 @@
 
   $masked-radius: mdc-shape-mask-radius($radius, 1 1 0 0);
 
-  &,
```

Comment:
We also need to apply shape to native control to keep the same shape as parent element.

Same below.

Reply:
Native control was removed (see comment below)